_posts/2018-05-29-Rust-1.26.1.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Announcing Rust 1.26.1"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.26.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.26.1 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.26.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1261-2018-05-29
+
+## What's in 1.26.1 stable
+
+A couple of issues were found in 1.26.0 which were deemed sufficient for a patch release.
+
+A quick summary of the changes:
+
+* RLS once again works properly on Windows
+* Rustfmt stopped badly formatting text in some cases
+* Returning from main via `impl Trait` where the Trait is not `Termination` is no longer permitted
+* `::<>` (turbofish) no longer works for method arguments whose type is `impl Trait`
+* `NaN > NaN` no longer returns true in const contexts
+* rustup should no longer fail due to missing documentation on some platforms
+
+If your code continues to compile, only the change to floating point
+comparisons may alter behavior.
+
+### RLS is fixed on Windows
```

Comment:
This is not quite right, it just happened that the problem was first reported on Windows. The issue is that RLS and command line builds were interfering (because they were inadvertently sharing a target directory) which meant that when switching between the two modes of compiling, you'd re-compile everything. On Windows this was made worse by the fact that there is a separate bug (which is not fixed) where either the compiler or the RLS is not releaseing locks on compiled proc macro libraries, which meant that compilation either freezes or crashes. The latter can still happen, but much less frequently.

Reply:
I've updated, could you take another look?